renderdoc/serialise/serialiser.h

```diff
@@ -592,7 +592,7 @@ class Serialiser
     }
 
     char *data = (char *)ReadBytes(sizeof(T));
-    f = *((T *)data);
+    memcpy(&f, data, sizeof(T));
```

Comment:
hmm OK, in this case I think it's necessary to #ifdef in the code rather than moving it to one of the OS files. Can you do something like:

```cpp

#if defined(_M_ARM) || defined(__arm__)
  memcpy(&f, data, sizeof(T));
#else
  f = *((T *)data);
#endif
```